src/test/java/api/item/ItemLastCheckInTests.java

```diff
@@ -133,14 +155,36 @@ public void shouldBeAbleCheckinItemWithoutLoanMultipleTimes() {
       .withRequestId("check-in-by-barcode-request")
       .withUserId(randomUserId);
 
-    loansFixture.checkInByBarcode(item, secondCheckInDate, servicePointId2,
+    loansFixture.checkInByBarcode(item, secondCheckInDateTime, servicePointId2,
       okapiHeaders);
 
     lastCheckIn = itemsClient.get(item.getId()).getJson()
       .getJsonObject("lastCheckIn");
 
-    assertThat(lastCheckIn.getString("dateTime"), is(secondCheckInDate.toString()));
+    actualCheckinDateTime = JsonPropertyFetcher
+      .getDateTimeProperty(lastCheckIn, "dateTime");
+
+    assertThat(actualCheckinDateTime, is(checkInDateTime));
     assertThat(lastCheckIn.getString("servicePointId"), is(servicePointId2.toString()));
     assertThat(lastCheckIn.getString("staffMemberId"), is(randomUserId));
   }
+
+  @Test
+  public void shouldDisplaySystemDateIfCheckinWasBackdated() {
+
+    IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    UUID servicePointId = servicePointsFixture.cd1().getId();
+    DateTime checkInDateTimeInPast = ClockManager.getClockManager().getDateTime()
+      .minusHours(1);
+
+    loansFixture.checkInByBarcode(item, checkInDateTimeInPast, servicePointId);
+    JsonObject lastCheckIn = itemsClient.get(item.getId()).getJson()
+      .getJsonObject("lastCheckIn");
+
+    DateTime actualCheckinDateTime = JsonPropertyFetcher
+      .getDateTimeProperty(lastCheckIn, "dateTime");
+
+    assertTrue(actualCheckinDateTime.isAfter(checkInDateTimeInPast));
```

Comment:
Why is this assertion needed?

Reply:
To show the case when the date in the past is supplied in the checkin.

Reply:
There is a more specific assertion below it, that checks the value of the actual check in date time.

What I don’t understand is why there is a need for both, when the second would inform the developer if it were incorrect.

Reply:
Oh I get it now. It's remains of the old approach. Removed.

Reply:
Thanks